### PR TITLE
Ga tuning

### PR DESCRIPTION
**Pull request by RialJohnson**

### CODE REVIEW

**Review by Rusty89 — APPROVED** [review]

File: Project 4/NeuralNets/src/Driver.java [hunk 1]
```diff
@@ -45,136 +50,164 @@ public static void main(String[] args) throws Exception {
...
         {
-            sum += result;
-            if (result < min)
-                min = result;
-            if (result > max)
-                max = result;
+            
+        }
+      
+        for (Network goodOne: best) {
```
Comment (GiantFrog) [inline Project 4/NeuralNets/src/Driver.java]:
```suggestion

```

File: Project 4/NeuralNets/src/Driver.java [hunk 2]
```diff
@@ -45,136 +50,164 @@ public static void main(String[] args) throws Exception {
...
+            System.out.println(goodOne);
```
Comment (GiantFrog) [inline Project 4/NeuralNets/src/Driver.java]:
```suggestion

```

File: Project 4/NeuralNets/src/Driver.java [hunk 3]
```diff
@@ -45,136 +50,164 @@ public static void main(String[] args) throws Exception {
...
         }
```
Comment (GiantFrog) [inline Project 4/NeuralNets/src/Driver.java]:
```suggestion

```

**Review by GiantFrog — CHANGES_REQUESTED** [review]
Just delete that stuff, it was a debugging thing I made

**Review by GiantFrog — APPROVED** [review]

**Pull request merged**